{{cookiecutter.project_shortname}}/assets/scss/{{cookiecutter.project_shortname}}/theme.scss

```diff
@@ -0,0 +1,11 @@
+/*
+ *   Copyright (C) 2020 CERN.
+ *   Copyright (C) 2020 Northwestern University.
+ *
+ * Invenio App RDM is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+@import "variables";
+@import "../invenio_app_rdm/theme";
+@import "../invenio_theme/theme";
```

Comment:
Should the `assets/` folder also be symlinked via `invenio-cli build --local` then (and the content transferred first) or did you have anothe workflow in mind?

Reply:
No it should be linked I just skipped that workflow after you finish the cli refactoring :)

Reply:
Well from what I understood with the discussion on gitter is that we won't go the symlinking route, and just copy things over. So my comment is moot and I will just copy this file over, so it's all good.

Reply:
I think that still assets folder should be linked otherwise the live reload won't work!

Reply:
Correct me if im wrong :) 

What I understood is that for now they wont be symlinked, until the watchpack/webpack hot reloading is fixed. Once that is working we would need to symlink to improve the dev experience.

Reply:
Now that I understand there is sort of 3 webpack assets locations: 

1- in each module, 
2- collected in the `<virtualenv>/var/instance/assets` and 
3- in the `<project_dir>/assets`

and that we are copying those from 3- into 2- , I don't know what the dev symlinking should be linking... But in any case it would be something to worry about later. For now: keep it simple, no symlinking.